### PR TITLE
update internal chain test script

### PR DESCRIPTION
**Pull request by area363**
## Update
- Add `delay_interval` parameter to customize tx staging interval:
image
- Ping slack channel when test is finished:
image

### CODE REVIEW

File: scripts/app/config.py [hunk 1]
```diff
@@ -13,6 +13,8 @@ class Config(NamedTuple):
     key_passphrase: Optional[str] = None
     # signer key address
     key_address: Optional[str] = None
+    # slack token
+    key_address: Optional[str] = None
```
Comment (eseiker) [inline scripts/app/config.py:17]: slack_token?

File: scripts/app/test_internal_chain.py [hunk 2]
```diff
@@ -87,3 +89,9 @@ async def process_block(blocks):
 
             # Increment the offset for the next query
             offset += current_limit
+
+        # The URL you want to send the data to
+        url = f'https://planetariumhq.slack.com/services/hooks/slackbot?token={config.slack_token}&channel=%239c-internal'
+        data = f"[9C-INFRA] Finished testing `{network}` network from `#{original_offset}` to `#{tip_index}`."
+        headers = {'Content-Type': 'text/plain'}
+        response = requests.post(url, data=data, headers=headers)
```
Comment (eseiker) [inline scripts/app/test_internal_chain.py:97]: should run only if config.slack_token exists
channel is hardcoded to 9c-internal

File: scripts/app/test_internal_chain.py [hunk 3]
```diff
@@ -22,19 +23,20 @@ def test(
...
         offset: Offset,
-        limit: Limit
+        limit: Limit,
+        delay: Delay
     ):
         # mainnet headless URL and initial setup
         mainnet_headless = "odin-full-state.nine-chronicles.com"
         internal_target_validator = "odin-internal-validator-5.nine-chronicles.com"
-        sleep_time = 5
+        sleep_time = delay
```
Comment (eseiker) [inline scripts/app/test_internal_chain.py:32]: just curious is it intended default sleep time reduced in odin-internal

**Review by sky1045 — APPROVED** [review]

File: scripts/app/test_internal_chain.py (hunk 3 above)
Comment (area363) [reply scripts/app/test_internal_chain.py:32]: Starting from this update, default sleep time is initially set to 0 for both networks(https://github.com/planetarium/9c-infra/pull/1511/files#diff-18621719fb04b6b3d0276ad0cffe51f2f8fd9bc3de1e51bb93ca00c6bd473349R22) and you can customize the interval upon each test.

File: scripts/app/test_internal_chain.py (hunk 2 above)
Comment (area363) [reply scripts/app/test_internal_chain.py]: I've updated the code to take the channel name as an environment variable 🙏

**Review by eseiker — APPROVED** [review]

**Pull request merged**